src/main/java/org/mskcc/cmo/metadb/irt_publisher/IRTUtil.java

```diff
@@ -81,7 +81,15 @@
         List<Map> requestListMap = mapper.readValue(requestListJSON, List.class);
         List<String> requestIds = new ArrayList<>();
         for (Map m : requestListMap) {
-            requestIds.add((String) m.get("requestId"));
+            String requestId = (String)m.get("requestId");
+            if (cmoRequestsOnly) {
+                boolean cmoRequest = (Boolean)m.get("isCmoRequest");
```

Comment:
field name should just be `cmoRequest`, not `isCmoRequest`. A little confusing since the request JSON delivers both of these but `isCmoRequest` will be going away.

Reply:
Is this for the Request Tracker or LimsRest?  I'm coding against the following endpoint/json:

https://igo.mskcc.org/request-tracker/swagger/#/RequestList/get_all

Reply:
oh gotcha, nvm then!